.eslintrc

```diff
@@ -2,7 +2,6 @@
   "extends": "airbnb",
   "parser": "babel-eslint",
   "rules": {
-    "brace-style": [2, "stroustrup"],
```

Comment:
this has nothing to do with the current PR, please try to provide PR as minimal as possible (futur stuff like this marked as SPR)